Nuget/Ploeh.AutoFixture.AutoFakeItEasy2.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>AutoFixture.AutoFakeItEasy2</id>
+        <version>@build.number@</version>
+        <title>AutoFixture with Auto Mocking using FakeItEasy</title>
+        <authors>Christopher Loggins</authors>
+        <owners>Christopher Loggins</owners>
+        <dependencies>
+            <dependency id="AutoFixture" version="@build.number@" />
+            <dependency id="FakeItEasy" version="2.0" />
+        </dependencies>
+        <summary>Turns AutoFixture into an Auto-Mocking Container based on FakeItEasy.</summary>
+        <description>This extension turns AutoFixture into an Auto-Mocking Container. The mock instances are created by FakeItEasy. To use it, add the AutoFakeItEasyCustomization to your Fixture instance. Read more at https://github.com/AutoFixture/AutoFixture.</description>
```

Comment:
Is there a better page that covers this? I only saw the docs on github (Cheat Sheet?), or your post about auto mocking. Which would be better?